### PR TITLE
Add helsinki harbors images

### PR DESCRIPTION
**Pull request by igordavydsson**
Refs #7

### CODE REVIEW

File: harbors/management/commands/add_helsinki_harbors_images.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+
+class Command(BaseCommand):
+
+    def handle(self, **options):
+        updated_harbors = 0
+        missing_images = []
+
+        for harbor in Harbor.objects.all():
+            with transaction.atomic():
```
Comment (tuomas777) [inline harbors/management/commands/add_helsinki_harbors_images.py]: There seems to be only one db affecting place (`harbor.image_file.save(...)`), so not sure if this atomic is actually doing anything in practice.

File: harbors/management/commands/add_helsinki_harbors_images.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+                self.stdout.write(
+                    'Assigning image for harbor with servicemap ID {}'.format(harbor.servicemap_id)
+                )
+                image_filename = '{}.jpg'.format(harbor.servicemap_id)
```
Comment (tuomas777) [inline harbors/management/commands/add_helsinki_harbors_images.py]: Should probably first check if `harbor.servicemap_id` is actually set? Or maybe better would be to loop only harbors that have `servicemap_id` set?

File: harbors/management/commands/add_helsinki_harbors_images.py (hunk 1 above)
Comment (igordavydsson) [reply harbors/management/commands/add_helsinki_harbors_images.py]: good point! will get rid of it

File: harbors/management/commands/add_helsinki_harbors_images.py (hunk 2 above)
Comment (igordavydsson) [reply harbors/management/commands/add_helsinki_harbors_images.py]: also good point! will fix this up

File: harbors/management/commands/add_helsinki_harbors_images.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+from harbors.models import Harbor
+
+
+class Command(BaseCommand):
+
+    def handle(self, **options):
+        updated_harbors = 0
+        missing_images = []
+
+        for harbor in Harbor.objects.filter(servicemap_id__isnull=False):
```
Comment (tuomas777) [inline harbors/management/commands/add_helsinki_harbors_images.py]: I still have a slight problem with this, as `servicemap_id` can be both `null` and `""`, I think more robust would be to check both of those.

Comment (igordavydsson) [reply harbors/management/commands/add_helsinki_harbors_images.py]: ok, had the same thought...

Comment (igordavydsson) [reply harbors/management/commands/add_helsinki_harbors_images.py]: fixed :)

**Pull request merged**